libs/utils/android/workloads/youtube.py

```diff
@@ -42,31 +40,46 @@ def __init__(self, test_env):
         # Set of output data reported by Jankbench
         self.db_file = None
 
-    def run(self, out_dir, collect,
-            video_url, video_duration_s):
+    def run(self, out_dir, video_url, video_duration_s, collect=''):
+        """
+        Run single YouTube workload.
+
+        :param out_dir: Path to experiment directory where to store results.
+        :type exp_dir: str
+
+        :param video_url: Video URL to be played
+        :type video_url: str
+
+        :param duration_s: Play video for this required number of seconds
+        :type duration_s: int
+
+        :param collect: Specifies what to collect. Possible values:
+            - 'energy'
```

Comment:
Why only energy?

There are not limitations from the Workload's module standpoint, which now supprots ftrace/systrace and energy:
https://github.com/ARM-software/lisa/blob/master/libs/utils/android/workload.py#L90

BTW thanks for the documentation! ;-)